### PR TITLE
Feature/pytorch

### PR DESCRIPTION
**Pull request by gabrielrizzo**
# ✨ Support for Pytorch model (Android Only)
- [Function] `setComputerVision`: Enables/disable computer vision model
- [Function] `setComputerVisionLoadModels`: Receives an array with paths.
- [Prop] `computerVision`: Enables/disable computer vision model

# 📝 Update Readme
- Updating props & functions docs.

### CODE REVIEW

File: src/vue.js [hunk 1]
```diff
@@ -107,7 +109,9 @@ export default {
         setFaceROIAreaOffset,
         setFaceROIAreaOffsetColor,
         setFaceContours,
-        setFaceContoursColor
+        setFaceContoursColor,
+        setComputerVision,
+        setComputerVisionLoadModels
```
Comment (TeruyaHaroldo) [inline src/vue.js:114]: These functions signatures must be in the `src/index.d.ts` too.

File: src/Yoonit.Camera.common.ts [hunk 2]
```diff
@@ -281,6 +285,8 @@ export abstract class CameraBase extends ContentView implements CameraDefinition
     public setFaceContours(enable: boolean) {}
 
     public setFaceContoursColor(color) {}
+
+    public setComputerVision(enable: boolean) {}
```
Comment (TeruyaHaroldo) [inline src/Yoonit.Camera.common.ts:289]: Missing the `setComputerVisionLoadModels` function in the `src/Yoonit.Camera.common.ts`.

**Review by TeruyaHaroldo — CHANGES_REQUESTED** [review]

File: demo-vue/app/components/App.vue [hunk 3]
```diff
@@ -202,6 +236,17 @@
       doPermissionDenied() {
         console.log('[YooCamera] doPermissionDenied');
       },
+
+      doVerifyMaskUsage(inferences) {
+          const TRASHOLD = 0.8
```
[the remark this answers is not in the document]
Comment (gabrielrizzo) [reply demo-vue/app/components/App.vue]: Done!

File: src/Yoonit.Camera.android.ts [hunk 4]
```diff
@@ -113,6 +113,21 @@ export class YoonitCamera extends CameraBase {
...
         this.nativeView.setFaceContoursColor(...color);
     }
+
+    @NativeMethod({ name: 'setComputerVision', length: 1 })
+    public setComputerVision(@Required enable: boolean) {
+        this.nativeView.setComputerVision(enable);
+    }
+
+    public setComputerVisionLoadModels(modelPaths: Array<String>) {
+        const nativeArrayList = new java.util.ArrayList()
```
[the remark this answers is not in the document]
Comment (gabrielrizzo) [reply src/Yoonit.Camera.android.ts]: Done

File: src/Yoonit.Camera.android.ts [hunk 5]
```diff
@@ -113,6 +113,21 @@ export class YoonitCamera extends CameraBase {
...
+
+        modelPaths.forEach((path) => {
+            nativeArrayList.add(path);
+        })
```
[the remark this answers is not in the document]
Comment (gabrielrizzo) [reply src/Yoonit.Camera.android.ts]: Done

File: src/Yoonit.Camera.common.ts (hunk 2 above)
Comment (gabrielrizzo) [reply src/Yoonit.Camera.common.ts:289]: It's only implemented on android and is not used as a prop setter. Should we create an empty function in these cases too?

Comment (gabrielrizzo) [reply src/Yoonit.Camera.common.ts:289]: Oh, i think I got it, now linter is warning that camera.common does not implement this method D:

File: src/vue.js (hunk 1 above)
Comment (gabrielrizzo) [reply src/vue.js:114]: Done

File: src/typings/android.d.ts [hunk 6]
```diff
@@ -53,6 +53,8 @@ declare namespace ai {
 					public setFaceROIEnable(param0: boolean): void;
 					public setFaceContours(param0: boolean): void;
 					public setFaceContoursColor(...params: number): void;
+					public setComputerVision(param0: boolean): void;
+					public setComputerVisionLoadModels(param0: any): void
```
[the remark this answers is not in the document]
Comment (gabrielrizzo) [reply src/typings/android.d.ts]: Done

File: src/Yoonit.Camera.android.ts [hunk 7]
```diff
@@ -113,6 +113,21 @@ export class YoonitCamera extends CameraBase {
     public setFaceContoursColor(@Required color) {
         this.nativeView.setFaceContoursColor(...color);
     }
+
+    @NativeMethod({ name: 'setComputerVision', length: 1 })
```
[the remark this answers is not in the document]
Comment (gabrielrizzo) [reply src/Yoonit.Camera.android.ts:118]: Done

**Review by TeruyaHaroldo — APPROVED** [review]

**Pull request merged**